### PR TITLE
VAMC-11201: VISN 1, VISN 5 and VISN 6 systems menus.

### PR DESCRIPTION
**Pull request by sundarka1**
## Description

Map out and Create Admin and Editorial Menus for VISN 1, 5 and 6 Systems.

## QA steps

As admin

- [x] Visit <Manage/content/menus/VAMC system menus> and validate that menus displayed match this
Screen Shot 2020-07-30 at 11 38 26 AM

Visit the following
- [x] /admin/structure/types/manage/health_care_local_facility
- [x] /admin/structure/types/manage/health_care_local_health_service
- [x]  /admin/structure/types/manage/health_care_region_page
- [x] /admin/structure/types/manage/regional_health_care_service_des
- [x] /admin/structure/types/manage/vamc_operating_status_and_alerts
and validate that the content types have new Menu items (VISN 1, 5 and 6 systems) selected as enabled (check box) to match this.
Screen Shot 2020-07-30 at 11 40 55 AM

- [x] Visit "Sections/veterans health administration/VAMC facilities" and validate that menus displayed for VISN 1, 5 and 6 match this.
Screen Shot 2020-07-21 at 2 54 39 PM
Screen Shot 2020-07-30 at 11 42 59 AM
Screen Shot 2020-07-30 at 11 44 08 AM

- [x] Visit "Manage/Content/Menus/CMS administration/Administration" and validate that VISN 1, 5 and 6 systems are assigned as child.
Screen Shot 2020-07-21 at 3 00 54 PM
Screen Shot 2020-07-30 at 11 46 43 AM
Screen Shot 2020-07-30 at 11 47 53 AM

- [x] Visit "/admin/config/user-interface/menu-breadcrumb" and validate VISN 1, 5 and 6 systems are enabled to match this.
Screen Shot 2020-07-30 at 11 51 21 AM

## Definition of Done
- [ ] Product release are written (or in progress), if required.
- [ ] Documentation has been updated, if applicable.

### CODE REVIEW

**Comment by kevwalsh** [comment]
@swirtSJW 

@sundarka1 @SgardnerPMP and @cmaeng and i just met, and i've added two tasks that should go in this PR (and all following PRs for future VISNs)

1) how to add enable "menu breadcrumb" for each new menu
2) how to update behat tests using Drupal Spec Tool for each new menu

This is all documented [here](https://docs.google.com/document/d/1fz-m_4Eo4Ib7g4s_3pk7Hz4N3uaNnCD1lAppp6KDMrk/edit?ts=5ee3aae7) 

/cc @davidconlon

**Comment by sundarka1** [comment]
> @swirtSJW
> 
> @sundarka1 @SgardnerPMP and @cmaeng and i just met, and i've added two tasks that should go in this PR (and all following PRs for future VISNs)
> 
> 1. how to add enable "menu breadcrumb" for each new menu
> 2. how to update behat tests using Drupal Spec Tool for each new menu
> 
> This is all documented [here](https://docs.google.com/document/d/1fz-m_4Eo4Ib7g4s_3pk7Hz4N3uaNnCD1lAppp6KDMrk/edit?ts=5ee3aae7)
> 
> /cc @davidconlon

File: config/sync/node.type.event.yml [hunk 1]
```diff
@@ -10,20 +10,28 @@ third_party_settings:
   menu_ui:
     available_menus:
       - va-altoona-health-care
+      - va-bedford-health-care
+      - va-boston-health-care
       - va-butler-health-care
+      - va-central-western-massuchesetts
```
Comment (swirtSJW) [inline config/sync/node.type.event.yml:16]: massachusetts?

File: config/sync/node.type.event_listing.yml [hunk 2]
```diff
@@ -11,20 +11,28 @@ third_party_settings:
     available_menus:
       - outreach-and-events
       - va-altoona-health-care
+      - va-bedford-health-care
+      - va-boston-health-care
       - va-butler-health-care
+      - va-central-western-massuchesetts
```
Comment (swirtSJW) [inline config/sync/node.type.event_listing.yml:17]: massachusetts

File: config/sync/core.menu.static_menu_link_overrides.yml [hunk 3]
```diff
@@ -593,5 +593,53 @@ definitions:
...
+    weight: 0
+    expanded: false
+    enabled: true
+  entity__menu__edit_form__va-boston-health-care:
+    menu_name: admin
+    parent: 'menu_link_content:9044de33-ae5e-4a36-bd57-4483ed84748e'
+    weight: 0
+    expanded: false
+    enabled: true
+  entity__menu__edit_form__va-central-western-massuchesetts:
```
Comment (swirtSJW) [inline config/sync/core.menu.static_menu_link_overrides.yml:608]: massachusetts

File: config/sync/node.type.health_care_local_facility.yml [hunk 4]
```diff
@@ -10,20 +10,28 @@ third_party_settings:
   menu_ui:
     available_menus:
       - va-altoona-health-care
+      - va-bedford-health-care
+      - va-boston-health-care
       - va-butler-health-care
+      - va-central-western-massuchesetts
```
Comment (swirtSJW) [inline config/sync/node.type.health_care_local_facility.yml:16]: massuchesetts

File: config/sync/node.type.health_care_region_page.yml [hunk 5]
```diff
@@ -10,20 +10,28 @@ third_party_settings:
   menu_ui:
     available_menus:
       - va-altoona-health-care
+      - va-bedford-health-care
+      - va-boston-health-care
       - va-butler-health-care
+      - va-central-western-massuchesetts
```
Comment (swirtSJW) [inline config/sync/node.type.health_care_region_page.yml:16]: massachusetts

File: config/sync/node.type.health_care_region_detail_page.yml [hunk 6]
```diff
@@ -11,20 +11,28 @@ third_party_settings:
     available_menus:
       - outreach-and-events
       - va-altoona-health-care
+      - va-bedford-health-care
+      - va-boston-health-care
       - va-butler-health-care
+      - va-central-western-massuchesetts
```
Comment (swirtSJW) [inline config/sync/node.type.health_care_region_detail_page.yml:17]: massachusetts

File: config/sync/node.type.health_services_listing.yml [hunk 7]
```diff
@@ -11,20 +11,28 @@ third_party_settings:
     available_menus:
       - outreach-and-events
       - va-altoona-health-care
+      - va-bedford-health-care
+      - va-boston-health-care
       - va-butler-health-care
+      - va-central-western-massuchesetts
```
Comment (swirtSJW) [inline config/sync/node.type.health_services_listing.yml:17]: massachusetts

File: config/sync/node.type.leadership_listing.yml [hunk 8]
```diff
@@ -11,20 +11,28 @@ third_party_settings:
     available_menus:
       - outreach-and-events
       - va-altoona-health-care
+      - va-bedford-health-care
+      - va-boston-health-care
       - va-butler-health-care
+      - va-central-western-massuchesetts
```
Comment (swirtSJW) [inline config/sync/node.type.leadership_listing.yml:17]: massachusetts

File: config/sync/node.type.locations_listing.yml [hunk 9]
```diff
@@ -11,20 +11,28 @@ third_party_settings:
     available_menus:
       - outreach-and-events
       - va-altoona-health-care
+      - va-bedford-health-care
+      - va-boston-health-care
       - va-butler-health-care
+      - va-central-western-massuchesetts
```
Comment (swirtSJW) [inline config/sync/node.type.locations_listing.yml:17]: massachusetts

File: config/sync/node.type.news_story.yml [hunk 10]
```diff
@@ -10,20 +10,28 @@ third_party_settings:
   menu_ui:
     available_menus:
       - va-altoona-health-care
+      - va-bedford-health-care
+      - va-boston-health-care
       - va-butler-health-care
+      - va-central-western-massuchesetts
```
Comment (swirtSJW) [inline config/sync/node.type.news_story.yml:16]: massachusetts

File: config/sync/node.type.person_profile.yml [hunk 11]
```diff
@@ -10,20 +10,28 @@ third_party_settings:
   menu_ui:
     available_menus:
       - va-altoona-health-care
+      - va-bedford-health-care
+      - va-boston-health-care
       - va-butler-health-care
+      - va-central-western-massuchesetts
```
Comment (swirtSJW) [inline config/sync/node.type.person_profile.yml:16]: massachusetts

File: config/sync/node.type.press_release.yml [hunk 12]
```diff
@@ -10,20 +10,28 @@ third_party_settings:
   menu_ui:
     available_menus:
       - va-altoona-health-care
+      - va-bedford-health-care
+      - va-boston-health-care
       - va-butler-health-care
+      - va-central-western-massuchesetts
```
Comment (swirtSJW) [inline config/sync/node.type.press_release.yml:16]: massachusetts

File: config/sync/node.type.press_releases_listing.yml [hunk 13]
```diff
@@ -11,20 +11,28 @@ third_party_settings:
     available_menus:
       - outreach-and-events
       - va-altoona-health-care
+      - va-bedford-health-care
+      - va-boston-health-care
       - va-butler-health-care
+      - va-central-western-massuchesetts
```
Comment (swirtSJW) [inline config/sync/node.type.press_releases_listing.yml:17]: massachusetts

File: config/sync/node.type.publication_listing.yml [hunk 14]
```diff
@@ -11,20 +11,28 @@ third_party_settings:
     available_menus:
       - outreach-and-events
       - va-altoona-health-care
+      - va-bedford-health-care
+      - va-boston-health-care
       - va-butler-health-care
+      - va-central-western-massuchesetts
```
Comment (swirtSJW) [inline config/sync/node.type.publication_listing.yml:17]: massachusetts

File: config/sync/node.type.story_listing.yml [hunk 15]
```diff
@@ -11,20 +11,28 @@ third_party_settings:
     available_menus:
       - outreach-and-events
       - va-altoona-health-care
+      - va-bedford-health-care
+      - va-boston-health-care
       - va-butler-health-care
+      - va-central-western-massuchesetts
```
Comment (swirtSJW) [inline config/sync/node.type.story_listing.yml:17]: massachusetts

File: config/sync/node.type.vamc_operating_status_and_alerts.yml [hunk 16]
```diff
@@ -10,20 +10,28 @@ third_party_settings:
   menu_ui:
     available_menus:
       - va-altoona-health-care
+      - va-bedford-health-care
+      - va-boston-health-care
       - va-butler-health-care
+      - va-central-western-massuchesetts
```
Comment (swirtSJW) [inline config/sync/node.type.vamc_operating_status_and_alerts.yml:16]: massachusetts

File: config/sync/system.menu.va-central-western-massuchesetts.yml [hunk 17]
```diff
@@ -0,0 +1,14 @@
...
+status: true
+dependencies:
+  module:
+    - workbench_menu_access
+third_party_settings:
+  workbench_menu_access:
+    access_scheme:
+      206: '206'
+id: va-central-western-massuchesetts
+label: 'VA Central Western Massuchesetts health care'
```
Comment (swirtSJW) [inline config/sync/system.menu.va-central-western-massuchesetts.yml:12]: massachusetts

File: tests/behat/drupal-spec-tool/menus.feature [hunk 18]
```diff
@@ -1,49 +1,57 @@
-@api
```
Comment (swirtSJW) [inline tests/behat/drupal-spec-tool/menus.feature:1]: Not all these should be showing as changed.  Only the lines that were modified by your work.
You may need to adjust your editor to not cleanup the pasted content from the spec tool.

Comment (sundarka1) [reply tests/behat/drupal-spec-tool/menus.feature:1]: I'm replaced the the menu.feature with the spectool output.

Comment (swirtSJW) [reply tests/behat/drupal-spec-tool/menus.feature:1]: Yes but when you do that, only the lines that changed, should show as changed in this diff.  Perhaps newline differences?

**Pull request merged**